app/contracts/contract.py

```diff
@@ -107,11 +107,12 @@ def call_function(contract: contract,
         _function = getattr(contract.functions, function_name)
 
         try:
+            _function = getattr(contract.functions, function_name)
             result = _function(*args).call()
-        except BadFunctionCallOutput:
+        except (BadFunctionCallOutput, ABIFunctionNotFound) as web3_exception:
             if default_returns is not None:
                 return default_returns
             else:
-                raise BadFunctionCallOutput
+                raise web3_exception
```

Comment:
Fix handling error in web3 wrapper with same way as ibet-Prime.
https://github.com/BoostryJP/ibet-Prime/blob/90f849bbdacce9b8d8600552c134785917a37d19/app/utils/contract_utils.py#L138-L159